tests/Examples/swift-bindings/core/basic.txt

```diff
@@ -0,0 +1,8 @@
+# Check basic 'core' functionality of the swift bindings
+#
+# REQUIRES: platform=Darwin
+# RUN: env TOOLCHAINS=swift xcrun -sdk macosx swiftc %{srcroot}/examples/swift-bindings/core/basic.swift -I %{srcroot}/build/products/swift-bindings -I %{srcroot}/products/libllbuild/public-api -Xlinker %{llbuild-lib-dir}/swift-bindings.dylib -o %t.exe
+# RUN: env LD_LIBRARY_PATH=%{llbuild-lib-dir} %t.exe %s > %t.out
+# RUN: cat %t.out
+# RUN: %{FileCheck} %s --input-file %t.out
+# CHECK: 6
```

Comment:
This commit is missing one thing, there needs to be a dependency from the `tests` target to the Swift bindings target, otherwise they don't get built before running the tests.